build_docs.pl

```diff
@@ -409,7 +409,18 @@ sub check_kibana_links {
         #
         # TODO: remove as part of
         # https://github.com/elastic/docs/issues/2264
-        $branch = $version eq "master" ? "main" : $version;
+        if ($version eq "master") {
+            $branch = "main"
+        }
+        else {
+            if ($version eq "8.19") {
+                $branch = "8.x"
```

Comment:
Should this be the reverse? The branch is `8.19` but the version we show in the dropdown etc is `8.x`?

Reply:
```suggestion
            if ($version eq "8.x") {
                $branch = "8.19"
```